### DISCUSSION TITLE
how to upgrade to latest build?

### DISCUSSION THREAD

**Discussion by ghost** [transmission/transmission#6921] [Q&A] [answered]
So if I run `sudo apt install transmission-daemon`
I get the following.
> transmission-daemon is already the newest version (4.0.5-1build5)

But the website: https://github.com/transmission/transmission/releases/
lists 4.0.6 as the latest. It says it's stable too.

But: https://launchpad.net/ubuntu/+source/transmission
lists 4.0.6 as active development
> [The Oracular Oriole](https://launchpad.net/ubuntu/oracular/+source/transmission) (active development)

if I want to just use apt install, how am I supposed to get 4.0.6?

**Comment by ghost** [comment]
is this community really this dead?

**Comment by Pentaphon** [comment, marked as answer]
You're on 24.04 and by using apt install you are restricted to the transmission versions for 24.04 which is 4.0.5. You have to use 24.10 test builds to use 4.0.6. 

Either update it manually or use Flatpak instead to get the latest version. 
https://transmissionbt.com/download
